scripts/check_chromium_revision.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2025 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import os
+import requests
+from time import time
+from google.cloud import storage
+
+DEFAULT_TIMEOUT = 600.0
+BUCKET_NAME = 'wpt-versions'
+NEW_REVISION_FILE = 'pinned_chromium_revision_NEW'
+OLD_REVISION_FILE = 'pinned_chromium_revision'
+PLATFORM_INFO = [
+    ("Win_x64", "chrome-win.zip"),
+    ("Win", "chrome-win.zip"),
+    ("Linux_x64", "chrome-linux.zip"),
+    ("Mac", "chrome-mac.zip")
```

Comment:
Outside the scope of this: But I wonder how this will work when we want Android given Chrome on Android runs on Chromium CI. We probably still want some coverage. Might be worth bringing this up to Weizhong.

Reply:
I'm not very familiar with the Chrome Android work, but is this shouldn't affect run for Chrome Android because it's considered a different product and does not use the pinned revision here (I might also be misunderstanding your comment 😅)